### DISCUSSION TITLE
Document stuck in queue - "poetry run flask db upgrade" command returned "None of PyTorch, TensorFlow >= 2.0, or Flax have been found"

### DISCUSSION THREAD

**Discussion by AcacioAraujo** [langgenius/dify#9577] [General]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### Content

Dify version
0.8.3

Installation method
Self-hosted using Docker

Steps to Reproduce
Tried to add knowledge base but document is stuck in queue
Worker log: mind attached file
![worker log](https://github.com/user-attachments/assets/c454e659-32e1-4936-9b55-fbd317362e2a)

tried to run "poetry run flask db upgrade" command, but returned message "None of PyTorch, TensorFlow >= 2.0, or Flax have been found. Models won't be available and only tokenizers, configuration and file/data utilities can be used."
mind attached file
![poetry log](https://github.com/user-attachments/assets/696f4136-461c-484b-97c2-5d9dec2aae19)

Expected Behavior
Document should be processed

Actual Behavior
document stuck in queue

**Comment by akira0912** [comment]
hello，is the problem solved? 你好，问题解决了吗？

**Reply by AcacioAraujo** [reply]
Yes, it is.
Thare was a docker compose file misconfiguration.
